### DISCUSSION TITLE
Using one cable adapter for Dreamcast 2 player or 4 player mode

### DISCUSSION THREAD

**Discussion by GrumpyGopher** [darthcloud/BlueRetro#774] [Q&A]
I'm looking to build a 4 player Dreamcast cable but would like to be able to disable ports 3 and 4.  I saw in the documentation that this involved connecting IO18 and IO23 to 3.3V to disable the player 3 and connecting IO26 and IO27 to 3.3V to disable the P4 port.  Since I was going to be cutting  some extension cables to build this I thought I would build a box with 2 Dreamcast ports that would take the 5v from the Dreamcast P3 and P4 controllers send it through an LDO to get the 3.3V and then send that to IO18, IO23, IO26, and IO27.  

Would this work or just blow everything up?

![image](https://github.com/darthcloud/BlueRetro/assets/27497732/9b49ee50-90b2-4ae4-9f65-4ee40ed9adb5)

**Comment by GrumpyGopher** [comment]
I did some testing with just two controllers connected, without sending pins to 3.3V, and it seemed to be working ok.  What kind of problems are you supposed to get?  Does it do random disconnects or something?

I also saw a different post that said to send IO18, IO23, IO26 and IO27 to ground instead of 3.3V.  Would that work?

**Comment by darthcloud** [comment]
The ESP32 is powered from your LDO 3V3 so it's totally fine to wire that 3V3 to the unused ESP32 pins.
